packages/components/src/DateTimePickers/MonthCalendar/MonthCalendar.scss

```diff
@@ -0,0 +1,10 @@
+
+.container {
+    display: inline-block;
+
+    .separator {
+        width: calc(100% - 20px);
+        border-color: grey;
```

Comment:
The mockups seems lighter. 
Use one of the color variables, see _colors.scss. Ask the ux the one to use, those are their vars.

Reply:
yep, i have the psd now. It was just for the look and feel temporary.